specification/resources/resource-manager/Microsoft.Authorization/stable/2020-09-01/examples/listDataPolicyManifests.json

```diff
@@ -0,0 +1,111 @@
+{
+  "parameters": {
+    "api-version": "2020-09-01"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Authorization/dataPolicyManifests/Microsoft.KeyVault.Data",
+            "type": "Microsoft.Authorization/dataPolicyManifests",
+            "name": "Microsoft.KeyVault.Data",
+            "properties": {
+              "namespaces": [
+                "Microsoft.KeyVault"
+              ],
+              "policyMode": "Microsoft.KeyVault.Data",
+              "isBuiltInOnly": true,
+              "resourceTypeAliases": [
+                {
+                  "resourceType": "vaults/certificates",
+                  "aliases": [
+                    {
+                      "name": "Microsoft.KeyVault.Data/vaults/certificates/keyProperties.keyType",
+                      "paths": [
+                        {
+                          "path": "keyProperties.keyType",
+                          "apiVersions": [
+                            "2019-01-01"
+                          ]
+                        }
+                      ]
+                    }
+                  ]
+                }
+              ],
+              "effects": [
+                {
+                  "name": "audit"
+                },
+                {
+                  "name": "deny"
+                }
+              ],
+              "fieldValues": [
+                "type"
+              ],
+              "resourceFunctions": {
+                "standard": [
+                  "subscription",
+                  "resourceGroup"
+                ],
+                "custom": [
+                  {
+                    "name": "vault",
+                    "fullyQualifiedResourceType": "Microsoft.KeyVault/vaults",
+                    "defaultProperties": [
+                      "location",
+                      "tags"
+                    ],
+                    "allowCustomProperties": false
+                  }
+                ]
+              }
+            }
+          },
+          {
+            "id": "/providers/Microsoft.Authorization/dataPolicyManifests/Microsoft.Kubernetes.Data",
+            "type": "Microsoft.Authorization/dataPolicyManifests",
+            "name": "Microsoft.Kubernetes.Data",
+            "properties": {
+              "namespaces": [
+                "Microsoft.Kubernetes",
+                "Microsoft.ContainerService"
+              ],
+              "policyMode": "Microsoft.Kubernetes.Data",
+              "isBuiltInOnly": false,
+              "effects": [
+                {
+                  "name": "EnforceOPAConstraint",
```

Comment:
@RamyasreeChakka is EnforceOPAConstraint effect ok? Or is it now deprecated?

Reply:
Please remove this effect from the example. But I do understand this effect is still returned in the GET dataPolicyManifests call We need to figure out a way how to show the deprecated effects and manifests to the users.